Source/ARTHttp.m

```diff
@@ -175,18 +179,19 @@ - (void)executeRequest:(NSMutableURLRequest *)request completion:(void (^)(NSHTT
     [self.logger debug:@"%@ %@", request.HTTPMethod, request.URL.absoluteString];
     [self.logger verbose:@"Headers %@", request.allHTTPHeaderFields];
 
+    __weak typeof(self) weakSelf = self;
```

Comment:
REST is released when the last request finishes. Realtime is release when the `close()` is called. @tcard Do you agree with the Realtime behaviour?